content/03.results.md

```diff
@@ -177,52 +171,50 @@ This trend may have resulted from therapy-induced mutations that produced additi
 
 #### Hypermutant tumors share mutational signatures and have dysregulated **_TP53_**
 
-We further investigated the mutational signature profiles of hypermutant (TMB > 10 Mut/Mb; N = 3) and ultra-hypermutant (TMB > 100 Mut/Mb; N = 4) tumors and/or derived cell lines from six patients in OpenPBTA (**Figure {@fig:Fig4}E**).
+We investigated the mutational signature profiles of hypermutant (TMB > 10 Mut/Mb; N = 3) and ultra-hypermutant (TMB > 100 Mut/Mb; N = 4) tumors and/or derived cell lines from six patients in OpenPBTA (**Figure {@fig:Fig4}E**).
 Five tumors were HGGs and one was a brain metastasis of a MYCN non-amplified neuroblastoma tumor.
 Signature 11, which is associated with exposure to temozolomide plus _MGMT_ promoter and/or mismatch repair deficiency [@doi:10.1038/s41588-019-0525-5], was indeed present in tumors with previous exposure to the drug (**Table {@tbl:Table2}**).
 We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with either constitutional mismatch repair deficiency (CMMRD) or Lynch syndrome (**Table {@tbl:Table2}**), genetic predisposition syndromes caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.1136/jmedgenet-2020-107627].
 Three of these patients harbored pathogenic germline variants in one of the aforementioned genes.
 While we did not detect a _known_ pathogenic variant in the germline of PT_VTM2STE3, this patient's pathology report contained a self-reported _PMS2_ variant, and we indeed found 19 intronic variants of unknown significance (VUS) in their _PMS2_.
 This is not surprising since an estimated 49% of germline _PMS2_ variants in patients with CMMRD and/or Lynch syndrome are VUS [@doi:10.1136/jmedgenet-2020-107627].
 Interestingly, while the cell line derived from patient PT_VTM2STE3's tumor at progression was not hypermutated (TMB = 5.7 Mut/Mb), it only contained the MMR2 signature, suggesting selective pressure to maintain a mismatch repair (MMR) phenotype _in vitro_.
-From patient PT_JNEV57VK, only one of the two cell lines derived from the progressive tumor was hypermutated (TMB = 35.9 Mut/Mb).
-This hypermutated cell line was strongly weighted towards signature 11, while this patient's non-hypermutated cell line showed several lesser signature weights (1, 11, 18, 19, MMR2; **Table S2**), highlighting the plasticity of mutational processes and the need to carefully genomically characterize and select models for preclinical studies based on research objectives.
+Only one of the two cell lines derived from patient PT_JNEV57VK's progressive tumor was hypermutated (TMB = 35.9 Mut/Mb).
+Their hypermutated cell line was strongly weighted towards signature 11, while their non-hypermutated cell line showed several lesser signature weights (1, 11, 18, 19, MMR2; **Table S2**).
```

Comment:
```suggestion
The hypermutated cell line was strongly weighted towards signature 11, while the non-hypermutated cell line showed several lesser signature weights (1, 11, 18, 19, MMR2; **Table S2**).
```

I would take out "their" as we prefer to separate the patient from the tumor or cell line in this instance. It is the cancer's genetics here reflected in the cell line, not the patient's.

Reply:
That answers my question in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/449#discussion_r1157826080.

Reply:
This change was implemented via a different suggestion - https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/449/commits/9a309829020d3362dabd60902e141b7ec314b5a1